marketplace/enrich.schema.json

```diff
@@ -9,24 +9,193 @@
             "additionalProperties": false,
             "properties": {
                 "weight": {
-                    "type": "boolean",
-                    "default": false
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "Enabled/disable weight enricher",
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "required": ["enabled"]
                 },
                 "category": {
-                    "type": "boolean",
-                    "default": false
+                    "type": "object",
+                    "additionalProperties": false,
```

Comment:
I didn't know the `additionalProperties` parameter. Thanks for helping me improve my knowledges. :-p